examples/jsm/renderers/webgl/nodes/WebGLNodeBuilder.js

```diff
@@ -274,9 +290,7 @@ class WebGLNodeBuilder extends NodeBuilder {
 
 		const varys = this.varys;
 
-		for ( let index = 0; index < varys.length; index ++ ) {
-
-			const vary = varys[ index ];
+		for ( const vary of varys ) {
 
 			snippet += `varying ${vary.type} ${vary.name}; `;
```

Comment:
I think this should be like this?

```js
const varyings = this.varyings;

for ( const varying of varyings ) {

    snippet += `varying ${varying.type} ${varying.name}; `;
```

Reply:
I updated here: https://github.com/mrdoob/three.js/pull/24388